modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/factory/CETFactoryImpl.java

```diff
@@ -279,6 +228,60 @@ public void validate(
 			oldCET);
 	}
 
+	@Activate
+	protected void activate() {
+		_cetImplFactories = HashMapBuilder.<String, CETImplFactory>put(
+			ClientExtensionEntryConstants.TYPE_COMMERCE_CHECKOUT_STEP,
+			new CommerceCheckoutStepCETImplFactoryImpl()
+		).put(
+			ClientExtensionEntryConstants.TYPE_CUSTOM_ELEMENT,
+			new CustomElementCETImplFactoryImpl()
+		).put(
+			ClientExtensionEntryConstants.TYPE_EDITOR_CONFIG_CONTRIBUTOR,
+			new EditorConfigContributorCETImplFactoryImpl()
+		).put(
+			ClientExtensionEntryConstants.TYPE_FDS_CELL_RENDERER,
+			new FDSCellRendererCETImplFactoryImpl()
+		).put(
+			ClientExtensionEntryConstants.TYPE_FDS_FILTER,
+			new FDSFilterCETImplFactoryImpl()
+		).put(
+			ClientExtensionEntryConstants.TYPE_GLOBAL_CSS,
+			new GlobalCSSCETImplFactoryImpl()
+		).put(
+			ClientExtensionEntryConstants.TYPE_GLOBAL_JS,
+			new GlobalJSCETImplFactoryImpl()
+		).put(
+			ClientExtensionEntryConstants.TYPE_IFRAME,
+			new IFrameCETImplFactoryImpl()
+		).put(
+			ClientExtensionEntryConstants.TYPE_JS_IMPORT_MAPS_ENTRY,
+			new JSImportMapsEntryCETImplFactoryImpl()
+		).put(
+			ClientExtensionEntryConstants.TYPE_STATIC_CONTENT,
+			new StaticContentCETImplFactoryImpl()
+		).put(
+			ClientExtensionEntryConstants.TYPE_THEME_CSS,
+			 new ThemeCSSCETImplFactoryImpl(_jsonFactory)
```

Comment:
@izaera do you see any concern in moving this to `activate()`)

Reply:
Just a note about this: `_jsonFactory` is null on the construct. It has something to do with the OSGI life cycle. By using the @Activate we make sure the dependencies are ready.

From my perspective, we don't have any issues with that

Reply:
Yeah, makes sense to move it to @activate